assets/src/scripts/web-services/nldi-data.spec.js

```diff
@@ -1,3 +1,8 @@
+import {MOCK_NLDI_UPSTREAM_FLOW_FEATURE, MOCK_NLDI_DOWNSTREAM_FLOW_FEATURE,
+    MOCK_NLDI_UPSTREAM_SITES_FEATURE,
+    MOCK_NLDI_DOWNSTREAM_SITES_FEATURE,
+    MOCK_NLDI_UPSTREAM_BASIN_FEATURE} from '../mock-service-data';
```

Comment:
This is a cool refactor - is the `mock-service-data` essentially a huge grab bag for all these tests now?

Reply:
Yes. In a few cases, I wanted to use the same mock data in more than one module so it seemed convenient to put the mock data in one place. I'm open to other ways of handling this.